index.html

```diff
@@ -5746,24 +5739,24 @@ <h4><dfn>Get Element Rect</dfn></h4>
 The returned value is a dictionary with the following members:
 
 <dl>
-<dt><dfn data-lt="elementrect-x">x</dfn>
+<dt>x
```

Comment:
We probably want to use `<code>` here as well. But this all might go into a different PR.